### DISCUSSION TITLE
Found A Source For Kotlin Icons In Java/Kotlin Manager.

### DISCUSSION THREAD

**Discussion by atikattar1104** [Sketchware-Pro/Sketchware-Pro#565] [Ideas]
> https://kotlinlang.org/docs/kotlin-brand-assets.html

This Is Official Source Of JetBrains Kotlin. As You Said In 'Known Issues' Of v6.4.0-RC4(Test Build 4), That, There Is No Icon For Kotlin File, This Might Be Helpful. But, We Need To Do Some Research, Whether Is It Ok To Use The Icon Directly, Or Do We Need Permission For That.

**Comment by j-px** [comment]
That's a good starting point, thanks!  
Just looked on Wikipedia for any logo usage terms, but found this instead:
![image](https://user-images.githubusercontent.com/40118124/188101535-b9e40cdd-fb02-4ef5-b30d-f1d88471740e.png)  
More about the trademark here: https://kotlinfoundation.org/guidelines/

So in the end, just the official logo as icon would be allowed.

**Reply by j-px** [reply]
Update: Now implemented with commit 29fa222eb3b159c3c58af1a61985e84f93bf070c (feat: Add icon to Kotlin files in Java/Kotlin Manager). If you want to, you can try it out by downloading this snapshot build: https://github.com/Sketchware-Pro/Sketchware-Pro/actions/runs/2978056167
